docs/legacy/connection.rst

```diff
@@ -0,0 +1,108 @@
+scrapinghub.legacy.Connection
+=============================
+
+TODO add short description & deprecation message here!
```

Comment:
or use http://www.sphinx-doc.org/en/stable/ext/todo.html#directive-todo for such notes